kubewarden-dashboard.json

```diff
@@ -1584,5 +1668,5 @@
   "timezone": "",
   "title": "Kubewarden",
   "uid": "r3Pw-1O7z",
-  "version": 5
+  "version": 2
```

Comment:
I guess this is not an issue right now :)

Reply:
:+1: